4.sc_Morphology_Neural_Network_MLP_Model/0.model_training/notebooks/Binary_Classifier_Call.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Consider adding some documentation in regards to your  `Binary_Classifier_Call.sh` in order for the user to know what is happening when using this script.

Reply:
An oversight on my part, I am fixing now.